### ISSUE TITLE
False positive on inconsistent indent when using CS 1.7.x chaining

### ISSUE THREAD

**Issue by emirotin** [clutchski/coffeelint#246]
```
module.exports.event = (req, res) ->
  slug = req.param 'slug'
  models.Event
    .findOne { slug }
    .populate 'organizer'
    .exec (err, event) ->
      if err # <--
        return res.send 500
      res.render 'event', event
```

False positive on the marked line — Line contains inconsistent indentation
I have invisible characters switched on and it shows I have consistent 2 spaces everywhere

I use coffeelint through Atom Editor package if that matters
